packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -1212,3 +1212,45 @@ describe('Auth service sends credentials fetch option to the ShopperLogin API',
         expect(shopperLogin.clientConfig.fetchOptions.credentials).toBe('same-origin')
     })
 })
+
+describe('clearECOMSession with hybridAuthEnabled', () => {
```

Comment:
Let's go with a slightly different description since `clearECOMSession with hybridAuthEnabled` made me think this was just a single test.

How about `hybridAuthEnabled property toggles clearECOMSession` in auth?